cluster/gce/config-test.sh

```diff
@@ -241,6 +241,17 @@ SCHEDULING_ALGORITHM_PROVIDER="${SCHEDULING_ALGORITHM_PROVIDER:-}"
 # Optional: install a default StorageClass
 ENABLE_DEFAULT_STORAGE_CLASS="${ENABLE_DEFAULT_STORAGE_CLASS:-true}"
 
+# Optional: Enable legacy ABAC policy that makes all service accounts superusers.
+if [[ "${E2E_UPGRADE_TEST:-}" == "true" ]]; then
```

Comment:
This value indicates that an upgrade step will run prior to a full e2e run. There are CI jobs that do upgrades that don't do this 2 pass behavior, so this will be unset for them.

Also, I'm not sure how closely we should tie the E2E* vars into this config. @fejta thoughts?